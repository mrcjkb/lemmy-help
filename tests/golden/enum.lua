---Experimental features that can be enabled
---@enum rocks.ExperimentalFeature
config.ExperimentalFeature = {
    --- Whether to install rocks stubs when using extensions
    --- like rocks-git.nvim or rocks-dev.nvim
    --- so that luarocks recognises them as dependencies
    ext_module_dependency_stubs = "ext_module_dependency_stubs",
    
    --- Some other experimental feature
    some_other_experimental_feature = "some_other_experimental_feature",
}

---@enum RustAnalyzerCmd
local RustAnalyzerCmd = {
  start = 'start',
  stop = 'stop',
  restart = 'restart',
  reload_settings = 'reloadSettings',
}

local U = {}

return U
