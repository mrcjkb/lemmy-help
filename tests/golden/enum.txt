rocks.ExperimentalFeature                            *rocks.ExperimentalFeature*
    Experimental features that can be enabled

    Values: ~
        ext_module_dependency_stubs       Whether to install rocks stubs when using extensions
                                          like rocks-git.nvim or rocks-dev.nvim
                                          so that luarocks recognises them as dependencies
        some_other_experimental_feature   Some other experimental feature


RustAnalyzerCmd                                                *RustAnalyzerCmd*

    Values: ~
        start
        stop
        restart
        reload_settings



